src/liboslexec/accum.cpp

```diff
@@ -130,10 +130,10 @@ AccumAutomata::accum(int state, const Color3 &color, std::vector<AovOutput> &out
 
 Accumulator::Accumulator(const AccumAutomata *accauto):m_accum_automata(accauto)
 {
-    const std::list<AccumRule> &rules = m_accum_automata->getRuleList();
+    const auto &rules = m_accum_automata->getRuleList();
     // Make sure we have as many outputs as the rules need
     int maxouts = 0;
-    for (std::list<AccumRule>::const_iterator i =  rules.begin(); i != rules.end(); ++i)
+    for (auto i = rules.cbegin(); i != rules.cend(); ++i)
         maxouts = i->getOutputIndex() > maxouts ? i->getOutputIndex() : maxouts;
```

Comment:
Range based iteration? Also the second line looks like an std::max

Reply:
There are LOTS more fixes we could do. I will tackle in a separate PR. See explanation I'm about to write.